helics/cli.py

```diff
@@ -305,8 +306,11 @@ def run(path, silent, connect_server, no_log_files, no_kill_on_error):
             if "env" in f:
                 for k, v in f["env"].items():
                     env[k] = v
+            p_args = shlex.split(f["exec"])
+            if p_args[0] == 'python':
+                p_args[0] = sys.executable
```

Comment:
I don't think this is an assumption we should have as a special case.

Try shutil.which() instead and see if that works.